app/models/concerns/presenters/statemachine/state_allows_single_child_creation.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Presenters::Statemachine
+  # Supports creation of a single child asset in this state
+  module StateAllowsSingleChildCreation
+    def control_additional_creation
+      yield if child_assets.blank?
```

Comment:
So the idea is that it only shows the button if current plate does not have children plate already created?

Reply:
Yep that's the one